lib/puppet_x/templates/init_ps.ps1.erb

```diff
@@ -391,6 +395,12 @@ function Reset-ProcessPowerShellVariables
     [Void]$ps.AddScript($ourFunctions)
     $ps.Invoke()
 
+    if ($WorkingDirectory -ne '') {
```

Comment:
The $WorkingDirectory in this case is always cast to a string due to Line 119 in powershell_manager.rb
```
...
WorkingDirectory = "#{working_dir}"
...
```